### DISCUSSION TITLE
Open multiple windows and switch using selenide?

### DISCUSSION THREAD

**Discussion by SkB1409** [selenide/selenide#1871] [Q&A]
How can I open a new window in selenide and switch in between the windows?

can someone help me out?

**Comment by AliakseiKarpuk** [comment]
Hi @SkB1409 you need to call new driver.

`SelenideDriver browser1 = new SelenideDriver(); `
`SelenideDriver browser2 = new SelenideDriver(); `
`browser1.open("https://google.com"); `
`browser2.open("http://yandex.ru");` 
`browser1.$(h1).shouldHave(text("Google"));` 
`browser2.$(h1).shouldHave(text("Yandeks"));`

See actual example [here](https://github.com/selenide/selenide/blob/master/src/test/java/integration/SelenideDriverITest.java)

**Comment by asolntsev** [comment]
@SkB1409  @AliakseiKarpuk The answer above is about two separate webdrivers. 
Probably a simpler solution would be to open a second window inside of the same webdriver:

```
  switchTo(newWindow(TAB));
  open("https://abother.url");
  switchTo(window(1));
```

**Reply by AliakseiKarpuk** [reply]
Thank you for your answer, but solution with two separate webdrivers more scalable or am I wrong?

**Reply by asolntsev** [reply]
Why it's scalable? What means scalable in the context of ui tests?

No, I don't see benefits in two browsers in this case.
